schema/types/acname_test.go

```diff
@@ -40,3 +40,21 @@ func TestNewACNameBad(t *testing.T) {
 		}
 	}
 }
+
+func TestSanitizeACName(t *testing.T) {
+	tests := map[string]string{
+		"foo#":              "foo-",
```

Comment:
Forgot to ask: can you please add one test-case of a name that _shouldn't_ be modified?